libs/ngxtension/signal-slice/src/signal-slice.spec.ts

```diff
@@ -138,11 +138,6 @@ describe(signalSlice.name, () => {
 			state.age();
 			expect(testFn).toHaveBeenCalled();
 		});
-
-		it('should connect lazy source after signal value is accessed', () => {
-			state().age;
```

Comment:
What is this a duplicate of?

The previous one is `state.age()` while this one is `state().age`

Reply:
Yeah. You are right and I should have paused checked again.